host.go

```diff
@@ -457,6 +478,7 @@ func (h *gpbftHost) collectChain(base ec.TipSet, head ec.TipSet) ([]ec.TipSet, e
 func (h *gpbftRunner) Stop(context.Context) error {
 	h.ctxCancel()
 	return multierr.Combine(
+		h.wal.Flush(),
```

Comment:
nit/tangent: rename Flush to Close (rotate)? not sure.

When I see "flush" I assume that the wal may not already be flushed, which defeats the whole crash-safety part. But writes _are_ synchronous, this just closes/rotates.